beaker-openstack.gemspec

```
@@ -17,6 +17,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.required_ruby_version = Gem::Requirement.new('>= 2.7')
```

You can write it simpler:
```suggestion
  s.required_ruby_version = '>= 2.7'
```

Though I'm always a fan of adding an `< 4` upper bound
```suggestion
  s.required_ruby_version = '>= 2.7', '< 4'
```